src/main/kotlin/no/nav/syfo/Bootstrap.kt

```diff
@@ -68,8 +69,6 @@ fun main() {
             }
 
             log.info("Hello from isprediksjon")
-
-            applicationState.ready = true
```

Comment:
Dette gjøres også i DatabaseModule som er et riktigere sted - sletter det derfor her slik at det ikke settes for tidlig.